scripts/taxonomy.R

```diff
@@ -0,0 +1,545 @@
+
```

Comment:
This file should be removed. As far as I'm concerned, we're no longer keeping any of the functions for checking taxonomy in the austraits.build

Reply:
Oops sorry. I had to put it back in temporarily and forgot to delete again